virttest/virt_vm.py

```diff
@@ -1380,6 +1380,7 @@ def wait_for_serial_login(self, timeout=LOGIN_WAIT_TIMEOUT,
             eg. during reboot or pause)
         :return: ConsoleSession instance.
         """
+        time.sleep(60) # Guest needs more time to start sometimes.
```

Comment:
this is going to affect and add every test a blind minute wait which would be costly when running tests in batches, I would say make it a controlled via user param and control it in your local test config and by default it will behaves as before without wait.

Reply:
OK, thanks